core/kazoo_ast/src/cb_api_endpoints.erl

```diff
@@ -42,7 +42,7 @@ api_to_ref_doc(Module, Paths, ?CURRENT_VERSION) ->
     DocPath = ?REF_PATH(BaseName),
     'ok' = file:write_file(DocPath, Doc);
 api_to_ref_doc(_Module, _Paths, _Version) ->
-    io:format("skipping ~p version ~p~n", [_Module, _Version]).
+    'ok'.
```

Comment:
Why remove such information? On the contrary I believe this is worth mentioning to the user: "no v1 endpoints end up in swagger.json".